pkg/types/replacement.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
```

Comment:
please move this file to sigs.k8s.io/kustomize/api/types/

Reply:
nevermind, did this already in another PR, please copy that instead to fix this PR.